test/test_helpers/storage.rb

```diff
@@ -87,7 +105,14 @@ def respond_to_missing?(m)
               attr_reader :inner
 
               def non_proxied_instances
-                ::ThreeScale::Backend::StorageAsync::Client.non_proxied_instances
+                klass = if inner.class == ThreeScale::Backend::StorageAsync::Client
```

Comment:
wouldn't this could be nicer as a switch/case?

Reply:
:+1: